src/coreclr/jit/llvm.cpp

```diff
@@ -588,7 +588,11 @@ const HelperFuncInfo& Llvm::getHelperFuncInfo(CorInfoHelpFunc helperFunc)
         { FUNC(CORINFO_HELP_PATCHPOINT) },
         { FUNC(CORINFO_HELP_CLASSPROFILE32) },
         { FUNC(CORINFO_HELP_CLASSPROFILE64) },
-        { FUNC(CORINFO_HELP_PARTIAL_COMPILATION_PATCHPOINT) }
+        { FUNC(CORINFO_HELP_PARTIAL_COMPILATION_PATCHPOINT) },
+
+        // TODO-LLVM: validating indirect calls, does LLVM need or want this?
+        { FUNC(CORINFO_HELP_VALIDATE_INDIRECT_CALL) },
+        { FUNC(CORINFO_HELP_DISPATCH_INDIRECT_CALL) }
```

Comment:
```suggestion
        { FUNC(CORINFO_HELP_PARTIAL_COMPILATION_PATCHPOINT) },
        { FUNC(CORINFO_HELP_VALIDATE_INDIRECT_CALL) },
        { FUNC(CORINFO_HELP_DISPATCH_INDIRECT_CALL) }
```
These `VALIDATE` helpers are part of the CFG feature which is [Windows-only](https://learn.microsoft.com/en-us/windows/win32/secbp/control-flow-guard).

It is possible (probable) that once/if we start thinking about doing non-WASM LLVM targeting we'll need it, but I believe at that point it'd be as simple as adding a flag for LLVM to generate the required code.